packages/application-shell/src/components/application-shell/application-shell.js

```diff
@@ -385,6 +385,8 @@ export default class ApplicationShell extends React.Component {
   static defaultProps = {
     trackingEventWhitelist: {},
   };
+  static version = process.env.npm_package_version;
```

Comment:
Btw, are the npm env variables always defined?

Reply:
Don't we need to use a rollup plugin to define this? like rollup plugin replace?

Reply:
No. It's a npm variable set. I also do the same in flopflip. More e.g. here https://github.com/facebook/create-react-app/issues/2466#issuecomment-312016096 or https://docs.npmjs.com/misc/scripts#packagejson-vars.

It's a good point that it would actual need interpolation/replacing.

Reply:
Also, those variables are only available if you run the process as a npm script, correct? I think we need to make it more safe, instead of relying on an "optional" environment variable.

WDYT?